src/tests/daos_perf.c

```diff
@@ -403,9 +404,13 @@ objects_verify_close(void)
 	int rc = 0;
 
 	if (ts_verify_fetch) {
-		rc = objects_verify();
-		fprintf(stdout, "Fetch verification: %s\n", rc ? "Failed" :
-			"Success");
+		if (ts_single || ts_overwrite) {
+			fprintf(stdout, "Verification is unsupported\n");
```

Comment:
There are two other things could be fixed:
- Explicitly notify user that 'overwrite' option only works in 'VOS' mode;
- Support 'T' in ts_val_factor();

Reply:
OK, I will disable it in another patch, I just realized I have to change a few other things in daos_perf